specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHubCluster.json

```diff
@@ -0,0 +1,591 @@
+              "required": [
+                "settings"
+              ]
+            },
+            "description": "Parameters for creating an Event Hubs Cluster resource."
+          }
+        ],
+        "x-ms-examples": {
+          "ClustersQuotasConfigurationPatch": {
+            "$ref": "./examples/Clusters/EHClusterQuotaConfigurationPatch.json"
+          }
+        },
+        "description": "Replace all specified Event Hubs Cluster settings with those contained in the request body. Leaves the settings not specified in the request body unmodified.",
+        "responses": {
+          "200": {
+            "description": "Event Hubs Cluster settings configuration returned.",
+            "schema": {
+              "$ref": "#/definitions/EHClusterQuotaConfigurationProperties"
+            }
+          },
+          "201": {
+            "description": "Event Hubs Cluster setting configuration request successfully accepted.",
+            "schema": {
+              "$ref": "#/definitions/EHClusterQuotaConfigurationProperties"
+            }
+          },
+          "202": {
+            "description": "Event Hubs Cluster update request accepted."
+          },
+          "default": {
+            "description": "Event Hubs Cluster error response describing why the get cluster quotas and settings operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Event Hubs Cluster Configuration"
+        ],
+        "operationId": "Clusters_ConfigurationGet",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "x-ms-examples": {
+          "ClustersQuotasConfigurationGet": {
+            "$ref": "./examples/Clusters/EHClusterQuotaConfigurationGet.json"
+          }
+        },
+        "description": "Get all Event Hubs Cluster settings - a collection of key/value pairs which represent the quotas and settings imposed on the cluster.",
+        "responses": {
+          "200": {
+            "description": "Event Hubs Cluster settings configuration returned.",
+            "schema": {
+              "$ref": "#/definitions/EHClusterQuotaConfigurationProperties"
+            }
+          },
+          "default": {
+            "description": "Event Hubs Cluster error response describing why the get cluster quotas and settings operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "EHCluster": {
```

Comment:
Looks like almost all the types have the "EH" prefix, is this really necessary?  For SDK content the types will already be in an eventhub namespace of some sort so the prefix seems redundant.